src/Microsoft.Azure.WebJobs.Host/Hosting/DrainModeManager.cs

```diff
@@ -85,5 +69,26 @@ public async Task EnableDrainModeAsync(CancellationToken cancellationToken)
                 _logger.LogInformation("Call to StopAsync complete, registered listeners are now stopped");
             }
         }
+
+        public void CancelFunctionInvocations()
+        {
+            foreach (var keyValuePair in _cancellationTokenSources)
+            {
+                string invocationId = keyValuePair.Key.ToString();
+                try
+                {
+                    _logger?.LogInformation($"Requesting cancellation for function invocation:{invocationId}");
```

Comment:
Same comments as the other logging one -- and call the key `functionInvocationId`